### PR TITLE
Connor rds boto3

### PR DESCRIPTION
**Pull request by connorjoleary**
Allow access to secrets and use that to log into db

### CODE REVIEW

File: backend/aws/lambda/app.py [hunk 1]
```diff
@@ -5,34 +5,65 @@
...
+
+def get_secret():
+    print('grabbing secret')
+    try:
+        get_secret_value_response = secret_client.get_secret_value(
+            SecretId=secret_name
+        )
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'DecryptionFailureException':
+            # Secrets Manager can't decrypt the protected secret text using the provided KMS key.
```
Comment (deadmau6) [inline backend/aws/lambda/app.py]: Should these comments also be printed out with the error?

Comment (connorjoleary) [reply backend/aws/lambda/app.py]: You are absolutely right, they should be

**Review by deadmau6 — APPROVED** [review]
:+1: seems good to me

File: backend/aws/lambda/app.py (hunk 1 above)
Comment (connorjoleary) [reply backend/aws/lambda/app.py]: I'm just going to remove them for now because they should be something you can just google.

**Pull request merged**